modules/todoitems.js

```diff
@@ -0,0 +1,166 @@
+import select2 from '../img/select.png';
+import delete2 from '../img/delete.png';
+import edit from '../img/edit.png';
+import save from '../img/save.png';
+
+export default class toDoItems {
+  constructor(text, completed, index) {
+    this.text = text;
+    this.completed = completed;
+    this.index = index;
+  }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222571020-f8d9469b-67f1-4390-b5ee-53ab1fac8870.png)

- [ ] It would be nice if you make your feature sounds like features to make your readme file descriptive and professional 😊

## Key Features
* Add a todo
* Remove a todo
* Update a todo